showcase/src/app/news/news-component/news.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+/* tslint:disable:completed-docs */
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { HttpClientTestingModule } from "@angular/common/http/testing";
+import { StoreModule } from "@ngrx/store";
+import { StarkLoggingService, STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+
+/**
+ * Load the implementations that should be tested.
+ */
+import { NewsComponent } from "./news.component";
+import SpyObj = jasmine.SpyObj;
+
+describe(`News`, () => {
+	let comp: NewsComponent;
+	let fixture: ComponentFixture<NewsComponent>;
+	let logger: SpyObj<StarkLoggingService>;
+
+	/**
+	 * async beforeEach.
+	 */
+	beforeEach(async(() => {
+		return (
+			TestBed.configureTestingModule({
+				declarations: [NewsComponent],
+				schemas: [NO_ERRORS_SCHEMA],
```

Comment:
Could you add a comment indicating which component/directive we don't want to import in this test as the reason why we use this schema?